sdk/spring/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/management/FeatureManager.java

```diff
@@ -310,6 +345,20 @@ private Mono<Boolean> isFeatureOn(FeatureFilterEvaluationContext filter, String
         return Mono.just(false);
     }
 
+    private TargetingFilterContext buildContext(Object appContext) {
+        if (contextualAccessor != null) {
+            TargetingFilterContext targetingContext = new TargetingFilterContext();
+            contextualAccessor.configureTargetingContext(targetingContext, appContext);
+            return targetingContext;
+        }
+        if (contextAccessor != null) {
+            TargetingFilterContext targetingContext = new TargetingFilterContext();
+            contextAccessor.configureTargetingContext(targetingContext);
+            return targetingContext;
+        }
+        throw new FeatureManagementException("No Targeting Filter Context found to assign variant.");
```

Comment:
Who will inject these two accessor? Our code or the customer's code? Because we don't seem to have default values for these two in our configuration.

Reply:
And are these two exclusive to each other?